### PR TITLE
[Merged by Bors] - chore: typo

### PR DESCRIPTION
**Pull request by ericrbg**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by mo271 — APPROVED** [review]

**Comment by alexjbest** [comment]
maintainer merge

**Review by eric-wieser — APPROVED** [review]
bors merge

File: docs/100.yaml [hunk 1]
```diff
@@ -385,7 +385,7 @@
 93:
   title  : The Birthday Problem
   links  :
-    mathlib archive: https://github.com/leanprover-community/mathlib4/blob/master/Archive/Wiedijk100Theorems/BirthdayProble.lean
+    mathlib archive: https://github.com/leanprover-community/mathlib4/blob/master/Archive/Wiedijk100Theorems/BirthdayProblem.lean
```
Comment (eric-wieser) [inline docs/100.yaml:388]: Ideally we would remove all these `mathlib archive` entries and switch to using `decls`, now that the archive is in doc-gen. That would catch all the typos, and allow linking to the final result.

Comment (ericrbg) [reply docs/100.yaml:388]: I can do this later today, if desired

**Pull request closed without merging**